index.js

```diff
@@ -220,7 +220,7 @@ Peer.prototype._addIceCandidate = function (candidate) {
     self._pc.addIceCandidate(
       new self._wrtc.RTCIceCandidate(candidate),
       noop,
-      function (err) { self._destroy(err) }
+      function (err) { location.reload(); self._destroy(err) }
```

Comment:
simple-peer also runs in Node, so location isn't appropriate here. Reload should be handled externally.